composer.json

```diff
@@ -23,7 +23,7 @@
         "php": ">=5.4.0",
         "symfony/framework-bundle": "~2.3|~3.0",
         "doctrine/doctrine-bundle": "~1.0",
-        "doctrine/migrations": "~1.0"
+        "doctrine/migrations": "dev-master"
```

Comment:
use ``~1.3@dev``, not ``dev-master``. This way, forgetting to change it won't hurt (as the stability flag is root-only anyway). This is the power of branch aliases